transcom/mymove
# B 17919 int

## B-17919

## UI - Update Zip to Use Address


## Verification Steps for Reviewers

These are to be checked by a reviewer.


### Setup to Run the Code

- [Instructions for starting storybook](https://transcom.github.io/mymove-docs/docs/frontend/setup/storybook)
- [Instructions for starting the MilMove application](https://transcom.github.io/mymove-docs/docs/getting-started/application-setup/)
- [Instructions for running tests](https://transcom.github.io/mymove-docs/docs/getting-started/development/testing)

### How to test

This test is to show that the full address fields now show for both Origin and secondary origin, and Destination and secondary destination. Secondly, the test will also show the checkbox functionality of populating the address fields when checked for both Origin and Destination sections.

1. Access the Client Application
2. Login as a Customer with a pre-existing incomplete ppm shipment.
3. In the Shipments section, click Edit to edit an existing ppm shipment.
4.  In the Origin section of the form, the whole address should now show. Click "Use current address".
5. An address should now display in most of the address fields.
6. Next under the address there should be text that reads, "Will you add items to your PPM from a different address?"  Click 'Yes'.
7. A secondary set of address fields should display.
8. In the Destination section of the form, a whole set of address fields should now show. Click "Use my current destination address".
9. An address should now display in most (but probably not all) of the address fields.
10. Next under the destination address fields there should be text that reads, "Will you deliver part of your PPM to a different address?"  Click 'Yes'.
11. A secondary set of address fields should display for the secondary destination.
